index.js

```diff
@@ -359,8 +360,14 @@ DumbPipe.registerOpener("camera", function(message, sender) {
     audio: false,
   }, function(localMediaStream) {
     mediaStream = localMediaStream;
+    url = URL.createObjectURL(localMediaStream);
 
-    video.src = URL.createObjectURL(localMediaStream);
+    video.onerror = video.onloadeddata = () => {
```

Comment:
That sounds more like a problem with the Firefox devtools, but I'll go ahead and remove this for now.